src/CircleGraph/CircleGraphCtrl.ts

```diff
@@ -145,27 +145,29 @@ export class CircleGraphCtrl {
   private registerEventHandlers() {
     // Handle messages from the webview
     this._webview.onDidReceiveMessage(message => {
+      switch (message.command) {
+        case MessageDefs.alert:
+          Balloon.error(message.text);
+          return;
+        case MessageDefs.request:
+          this.handleRequest(message.url, message.encoding);
+          return;
+        case MessageDefs.loadmodel:
+          this.handleLoadModel(parseInt(message.offset));  // to number
+          return;
+        case MessageDefs.finishload:
+          this.handleFinishLoad();
+          return;
```

Comment:
At first, I was to recommend to add `default` but it prevents going to `handleReceiveMessage` function.
So I wonder why only some of commands are relocated, not all of them :)

Reply:
Thanks for the comment :)
Relocated message have no or little relation with event handlers.
My direct target is to simplify `interface CircleGraphEvent` so that this won't depend on individual message types changes. The whole change + others exist in #1351 but maybe hard to read -_-;

Reply:
I see it :)
Then could you add following comments? Just not to forget..!

```suggestion
          return;
        // default:
        //   TODO Handling undefined messages
```

Reply:
As below `this.handleReceiveMessage(message);` is called and it will stay, there will be no `default` handling.

Reply:
I've added `default` case for fall through.